CHANGES.md

```diff
@@ -26,7 +26,11 @@ Currrent Changes
 
 - Remove support for EGD random source.
 
-- Made the ERR_STACK datatype opaque.
+- Made the `ERR_STACK` datatype opaque.
+
+- Removed some functions deprecated in the OpenSSL 1.1.0 release:
+`SRP_VBASE_get_by_user`, `ENGINE_setup_bsd_cryptodev`,
+`PKCS12_get_attr`, and `SSL_set_debug`.
```

Comment:
Should the previous two lines be indented by two spaces to make them line up with secondary bullet lists

Reply:
I think the way it is now, it's just one single paragraph, which is okay.  Rather than edit/format and go through the whole CI pipeline again. :)